packages/core/src/components/graphs/scatter.ts

```diff
@@ -64,6 +77,9 @@ export class Scatter extends Component {
 				);
 		}
 
+		// filter out of domain data
+		scatterData = this.filterOutOfDomain(scatterData);
```

Comment:
is this something that needs to live in the `Scatter` component?

Reply:
Yes, we may see partial circle at the edge in some charts.
![image](https://user-images.githubusercontent.com/59426533/87134104-c99e5280-c2ca-11ea-99ce-4bc52ebec3d1.png)